blocks/block-alignment-toolbar/index.js

```diff
@@ -59,8 +55,8 @@ export function BlockAlignmentToolbar( { value, onChange, controls = DEFAULT_CON
 	);
 }
 
-export default withEditorSettings(
-	( settings ) => ( {
-		wideControlsEnabled: settings.alignWide,
+export default withSelect(
+	( select ) => ( {
+		wideControlsEnabled: select( 'core/editor' ).getEditorSettings().alignWide,
```

Comment:
Targeted selectors like `doesEditorSuppoortWideAlignment`

Reply:
What about having an option to enabling this support to the existing selector?

```js
wideControlsEnabled: select( 'core/editor' ).getEditorSettings( 'alignWide' ),
```